docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| ⚠️ **Alerts**                  | Available                                                              |
+| 📊 **Normals**                 | Available                                                              |
+| 🧭 **Reverse geocoding**       | Available: will show the name of the nearest location                  |
+
+For the United States, [Forecast Advisor](https://www.forecastadvisor.com/) has temperature and precipitation 1-3 days accuracy comparison by city between the following sources: **Accuweather**, [NWS](#national-weather-service), [Open-Meteo](#open-meteo) and [Pirate Weather](#pirate-weather).
+
+<details><summary><h4>Details of available data from Accuweather</h4></summary>
+
+| Data                      | Available | Data              | Available   |
+|---------------------------|-----------|-------------------|-------------|
+| Weather Condition         | ✅         | Pressure          | ✅ (Current) |
+| Temperature               | ✅         | UV Index          | ✅           |
+| Precipitation             | ✅ (RSI)   | Sunshine Duration | ✅           |
+| Precipitation Probability | ✅ (TRSI)  | Sun &amp; Moon    | ✅           |
+| Precipitation Duration    | ✅ (RSI)   | Moon Phase        | ✅           |
+| Wind                      | ✅         | Cloud Cover       | ✅           |
+| Humidity                  | ✅         | Visibility        | ✅           |
+| Dew Point                 | ✅         | Ceiling           | ✅           |
+</details>
+
+### OpenWeather
+**[OpenWeather](https://openweathermap.org/)** is a weather data provider based in London, United Kingdom.
+
+| Feature                        | Detail                                                                                                                                                                                                                           |
+|--------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 🔐 **API key**                 | Optional: Bundled API key is often rate-limited. You can configure your own API key, however OpenWeather asks for credit card information even if you only want to use the free tier. [Register](https://openweathermap.org/api) |
+| 🗺️ **Coverage**               | 🌐 Worldwide (some features may not be available for all locations)                                                                                                                                                              |
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station                                                                                                                                                           |
```

Comment:
Extend my previous comment to everywhere you noted this. Apart from national sources which have their own stations, this is rarely documented if they use stations